docs/platform/howto/project-support-center.rst

```diff
@@ -29,10 +74,70 @@ To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
     * Any error messages
     * Any languages or frameworks you are using
 
-#. Select the affected project, service, and a severity level.
+#. Optional: Upload files such as screenshots, logs, or :ref:`HAR files <create-har-files>`.
+   
+   .. important::
+        Aiven support will never ask you to provide sensitive data such as passwords or personal information. You should remove (not redact) any sensitive data in files that you attach to a support ticket.
 
 #. Click **Create ticket**. 
 
-The new support ticket is sent to our support team for review and you also receive a confirmation email. `Response times <https://aiven.io/support-services>`_ vary by case severity and support tier. 
+If you are on a paid support tier, your account team is also notified when you submit a ticket. 
```

Comment:
```suggestion

```
I don't think we should be saying it in public and I doubt all tickets will be monitored by the account teams.